src/docfx/build/Build.cs

```diff
@@ -266,5 +286,29 @@ private static IReadOnlyList<Document> GetTableOfContentsScope(Docset docset, Ta
 
             return result;
         }
+
+        private class SourceRepo
```

Comment:
we already have a `Repository` class, can that be reused?

Reply:
`Repository` need have "path"